app/helpers/users_helper.rb

```diff
@@ -1,2 +1,9 @@
 module UsersHelper
+    # Returns the Gravatar for the given user.
+  def gravatar_for(user, size: 80)
```

Comment:
Whatever you have done here still does not solve the Single responsibility issue I had earlier.
extract a method that returns the gravatar_url then pass the url to the image_tag